commercetools/importer/commercetools-api-default/src/test/java/cleanup/DeleteEverythingIntegrationTest.java

```diff
@@ -273,23 +270,6 @@ private void deleteAllChannels() {
         } while (response.getResults().size() != 0);
     }
     
-    private void deleteAllApiClients() {
```

Comment:
this drove me crazy, it deleted all API's not named main 😞 , and i would just see my api client disappear and the tests fail.
i could fix it, but taught it doesn't make sense so i removed it totally

Reply:
I see and if I remeber it correctly, I had a discussion with Alex about it too 😉